src/main/java/au/com/agiledigital/idea_search/blueprints/BlueprintPageContextProvider.java

```diff
@@ -41,6 +39,7 @@
    */
   private String renderValue(KeyProperty property) {
     if (property.key.startsWith("v")) {
+      String templatePath = "vm/";
       StringBuilder builder = new StringBuilder(templatePath);
```

Comment:
Would this be better if the "vm/" was straight into the parameter
```suggestion
      StringBuilder builder = new StringBuilder("vm/");
```

Reply:
`templatePath` is used on line 63 otherwise yes.